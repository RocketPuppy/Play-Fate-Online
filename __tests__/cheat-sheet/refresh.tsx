import * as React from "react";
import * as renderer from "react-test-renderer";
import Refresh from "../../src/cheat-sheet/refresh";

test("renders all the pieces", () => {
  const render = renderer.create(<Refresh />);

  expect(render.toJSON()).toMatchSnapshot();
});
