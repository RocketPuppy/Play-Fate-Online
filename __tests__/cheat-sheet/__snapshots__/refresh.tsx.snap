// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Refresh
  </h2>
  <p>
    At the start of a new session, you reset your fate points to your refresh rate. If you ended the last session with more points, you keep the extra. At the end of a scenario, you reset to your refresh rate no matter what.
  </p>
</div>
`;
